searx/settings.yml

```diff
@@ -1076,10 +1083,12 @@ engines:
   - name: qwant
     engine: qwant
     shortcut: qw
-    categories: general
+    categories: [general, web]
     disabled: false
     additional_tests:
       rosebud: *test_rosebud
+    about:
+      powered_by: Bing
```

Comment:
Isn't it the same for the other qwant engines ('qwant images', 'qwant news' ...)?

Are there any reasons why "powered_by" is sometimes set in the settings.yml and sometimes direct in the engine (aka module)?

Reply:
>Isn't it the same for the other qwant engines ('qwant images', 'qwant news' ...)?

For web engines it's easy to verify by searching for a site that returns the User-Agent. For images and news it's not so easy to verify.

>Are there any reasons why "powered_by" is sometimes set in the settings.yml and sometimes direct in the engine (aka module)?

Yes it follows the existing convention of defining metadata in the module if possible (e.g. categories are currently defined in the engines when possible).

Reply:
> For web engines it's easy to verify by searching for a site that returns the User-Agent. For images and news it's not so easy to verify.

I do not understand this answer .. perhaps I do not understand what "powered by" means .. I thought, the results of this engine (e.g. qwant) are powered by e.g. Bing .. I am wrong? .. what do you mean "User-Agent"?

> Yes it follows the existing convention of defining metadata in the module if possible (e.g. categories are currently defined in the engines when possible).

I asked, because you defined "powered by" of qwant in settings.yml but in bing (itself) and ddg you defined it in the engine module:

![grafik](https://user-images.githubusercontent.com/554536/148056042-660f77d0-c2f4-451e-8887-ce3b041fc96e.png)

Reply:
>I thought, the results of this engine (e.g. qwant) are powered by e.g. Bing .. I am wrong? .. what do you mean "User-Agent"?

Search engines like Qwant don't actually disclose in detail which results are powered by what. One way to find out is to find the User-Agent of the result crawler in their search result descriptions.

>I asked, because you defined "powered by" of qwant in settings.yml but in bing (itself) and ddg you defined it in the engine module

Yes because I wanted to avoid categorizing Qwant Video & Images with powered by since I actually don't know that.

Reply:
Qwant is powerd by Bing .. but both have different results for the same search term ... 

TBH we do not know which engine is "powered by" which search engine .. AFIAK qwant mix results from bing and from a qwant index.

No matter from which point I look on this "powered by" .. it should never be a sort criteria, since it could never be a serious criteria.

Reply:
> it should never be a sort criteria, since it could never be a serious criteria

Of course it can (notice the bingbot):

![image](https://user-images.githubusercontent.com/73739153/148066210-2aecbc4d-441a-4948-9010-b3459eb7740c.png)

Reply:
>  (notice the bingbot):

Sorry if I ask a dumb question, but how reliable is the assumption that ddg uses (only) results from a Bing partnership?

Reply:
[from their website](https://help.duckduckgo.com/duckduckgo-help-pages/results/sources/):

>We also of course have more traditional links in the search results, which we also source from multiple partners, **though most commonly from Bing**